src/main/java/org/mamute/model/Tag.java

```diff
@@ -57,12 +64,7 @@ public String getName() {
 	}
 	
 	public String getUriName() {
-		try {
-			return URLEncoder.encode(name, "utf-8");
-		} catch (UnsupportedEncodingException e) {
-			e.printStackTrace();
-		}
-		return null;
+		return this.sluggedName;
```

Comment:
Could you test tags like `c++` and `c#` please? The slugged name cannot be just `c` since it would conflict